CHANGELOG.md

```diff
@@ -5,42 +5,43 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
-- Add new configuration parameter `Scheduling/SlurmSettings/QueueUpdateStrategy` to allow cluster update when
-  `SlurmQueues` configuration changes don't impact Slurm scheduler configuration.
-- Add support for multiple Elastic File Systems.
-- Add support for multiple FSx File Systems.
-- Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
-- Add support for FSx Lustre Persistent_2 deployment type.
-- Add support for memory-based scheduling in Slurm.
-  - Configure `RealMemory` on compute nodes by default as 95% of the EC2 memory.
-  - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
+- Add support for memory-based job scheduling in Slurm
+  - Configure compute nodes real memory in the Slurm cluster configuration.
+  - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to enable memory-based scheduling in Slurm.
   - Add new configuration parameter `Scheduling/SlurmQueues/ComputeResources/SchedulableMemory` to override default value of the memory seen by the scheduler on compute nodes.
+- Improve flexibility on cluster configuration updates to avoid the stop and start of the entire cluster whenever possible.
+  - Add new configuration parameter `Scheduling/SlurmSettings/QueueUpdateStrategy` to set the preferred strategy to adopt for compute nodes needing a configuration update and replacement.
+- Add support to mount existing FSx for ONTAP and FSx for OpenZFS file systems.
+- Add support to mount multiple instances of existing EFS, FSx for Lustre / for ONTAP/ for OpenZFS file systems.
+- Add support for FSx Lustre Persistent_2 deployment type.
```

Comment:
nit: might clarify that this is for created filesystems.

Reply:
Addressed this comment, thanks 👍